runtime/src/snapshot_package.rs

```diff
@@ -197,9 +175,12 @@ impl AccountsPackage {
     /// NOTE 2: This fn will panic if the AccountsPackage is of type EpochAccountsHash.
```

Comment:
Cannot comment outside of line range....

NOTE 1 is now wrong. It's not within a `TempDir`

Reply:
Removed NOTE 1

Reply:
Removed NOTE 1